cnd/src/swap_protocols/rfc003/ethereum/htlc_events.rs

```diff
@@ -80,9 +80,14 @@ impl
         htlc_params: &HtlcParams<Ethereum, asset::Ether, identity::Ethereum>,
         start_of_swap: NaiveDateTime,
     ) -> anyhow::Result<Deployed<htlc_location::Ethereum, transaction::Ethereum>> {
+        let expected_bytecode = htlc_params.bytecode();
+
         let (transaction, location) =
-            watch_for_contract_creation(self, start_of_swap, htlc_params.bytecode())
-                .instrument(tracing::info_span!("htlc_deployed"))
+            watch_for_contract_creation(self, start_of_swap, &expected_bytecode)
+                .instrument(tracing::trace_span!(
```

Comment:
Please record an issue to audit all uses of `xy_span!` to set values on the right level, with the result of likely creating more than one span :)

More information is found in point 4 of this post: https://github.com/tokio-rs/tracing/issues/442#issue-526689793

Reply:
Can do.

Reply:
https://github.com/comit-network/comit-rs/issues/2407